kubernetes/ama-logs.yaml

```diff
@@ -913,8 +911,6 @@ spec:
         dockerProviderVersion: "18.0.1-0"
         schema-versions: "v1"
         kubernetes.azure.com/no-http-proxy-vars: "true"
-        # prometheus.io/scrape: "true"
```

Comment:
any specific reason this change is part of this PR?

Reply:
> any specific reason this change is part of this PR?

This is just clean  up. This is initially added since prom addon didnt had support without annotation. we dont use these for AKS RP yaml.